src/lib/documents.ts

```diff
@@ -72,3 +112,64 @@ export function isNIT(str: string): boolean {
 
   return Number(verifier) === sum;
 }
+
+/**
+ * Verifies that given NIT with hyphens is valid
+ *
+ * @param  {string} str       A string representing NIT digits (with hyphens)
+ * @returns {boolean}         Validity of the given NIT
+ */
+function isNITWithHyphen(str: string): boolean {
+  if (!nitRegExp.test(str)) return false;
+  const [municipality, birthdate, correlative, verifier] = str.split('-');
+  return isValidNIT(municipality, birthdate, correlative, verifier);
+}
+
+/**
+ * Verifies that given NIT in numeric format is valid
+ *
+ * @param  {string} str       A string representing NIT digits (without hyphens)
+ * @returns {boolean}         Validity of the given NIT
+ */
+function isNITWithNumbers(str: string): boolean {
+  if (!(str.length === 14)) return false;
```

Comment:
Can you please:

1. Submit this feature in a separate PR
2. Evaluate the string using RegExp, instead of just validating the length of the string. In this way, we are going to be able to verify that the string only contains digits